pkg/customer/request.go

```diff
@@ -0,0 +1,41 @@
+package customer
+
+import "time"
+
+// Request represents a request for creating a customer.
+type Request struct {
+	DefaultAddress string `json:"default_address,omitempty"`
+	DefaultCard    string `json:"default_card,omitempty"`
+	Description    string `json:"description,omitempty"`
+	Email          string `json:"email,omitempty"`
+	FirstName      string `json:"first_name,omitempty"`
+	LastName       string `json:"last_name,omitempty"`
+
+	DateRegistered *time.Time            `json:"date_registered,omitempty"`
+	Address        AddressResponse       `json:"address,omitempty"`
```

Comment:
Precisamos definir as structs que representam objetos de request como ponteiro para evitar serializar valores default. 
```suggestion
	Address        *AddressResponse       `json:"address,omitempty"`
```